src/uu/numfmt/src/numfmt.rs

```diff
@@ -465,9 +472,9 @@ mod tests {
         let result_display = format!("{result}");
         assert_eq!(
             result_debug,
-            "FormattingError(\"invalid suffix in input: 'hello'\")"
+            "FormattingError(\"numfmt-error-invalid-suffix\")"
         );
-        assert_eq!(result_display, "invalid suffix in input: 'hello'");
+        assert_eq!(result_display, "numfmt-error-invalid-suffix");
```

Comment:
This looks incorrect.

Reply:
actually, it is, the translation engine initialized.
I could initialized it but it seems overkill :)